frontend/public/components/monitoring/targets.tsx

```diff
@@ -31,9 +37,73 @@ import { Labels } from './labels';
 import { AlertSource, PrometheusAPIError, Target } from './types';
 import { targetSource } from './utils';
 
+enum MonitorType {
+  SERVICEMONITOR = 'serviceMonitor',
+  PODMONITOR = 'podMonitor',
+  UNDEFINED = 'undefined-type',
+}
+
+const FindMonitorType = (target: Target): MonitorType => {
```

Comment:
ok, modified.

Reply:
I'll remove this whole function since we're doing the operation in each functional component. 
(i.e. `const isServiceMonitor: boolean = target.scrapePool.includes(MonitorType.ServiceMonitor);`
  `const isPodMonitor: boolean = target.scrapePool.includes(MonitorType.PodMonitor);`)